user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+convince them to complete the checkout and place an order. You can find more details about this functionality in the
+:ref:`Magento Abandoned Cart Campaigns guide <user-guide-acc>`.
+
+This functionality is useful for marketing representatives and any other employees who are working with email campaigns 
+directed at customers of Magento-based store.
+
+.. important::
+
+    To use the functionality, you also need to :ref:`set up the Abandoned Cart extension <user-guide-acc-ext>`.
+
+    
+.. _admin-capabilities-data-audit:    
+    
+Data Audit
+----------
+
+If this capability is set to *"System"*, the user can see the full history of changes made to any record of an
+:ref:`auditable entity <user-guide-entity-management-create-other>`, as well as an out-of-the-box report of all such 
+actions. 
+
+The report is available from the "*System*" menu and described in more details in the 
+:ref:`Data Audit <user-guide-data-audit>` guide.
+
+The link to a specific record's history of changes is available in the top right corner of the record’s
+:ref:`View page <user-guide-ui-components-view-pages>`.
+
+|
+
+.. image:: ./img/data_management/view/view_history.png
+
+|
+
+This Data Audit is usually used by system administrators, but may also be used by managers in order to see how details 
+of different records were changed, as well as by whom.
+
+
+.. _admin-capabilities-dotmailer-stats:    
+
+Dotmailer Statistic
+-------------------
+
+This capability is meaningful if the system has been integrated with dormatiler. 
+
+Such integration lets users utilize dotmailer as an email campaign engine, as described in the guide on
+:ref:`Integration with Dotmailer <user-guide-dm-integration>`. So, the user can synchronize lists of contacts in OroCRM 
+and dotmailer, and use them to create email campaigns in dotmailer and import them to OroCRM. 
+
+If the *Dotmailer Statistic* capability has been set to *System*, the user will be able to see the dotmailer campaign 
+statistics to analyze the efficiency of the campaign. 
+
+These statistics are usually used by marketing managers and representatives responsible for email campaigns.
+
+.. _admin-capabilities-export-entities:    
+
+Export Entities
+---------------
+
+If this capability is set to *"System"*, the user can export an entity's records into a .CSV file, as 
+described in the :ref:`Import and Export Data <user-guide-export>` guide. 
+
+The :guilabel:`Export` button will appear in the top right corner of the page.
+
+|
+
+.. image:: ./img/roles/export.png 
+
+|
+
+Export is a general productivity tool that is usually enabled for most users.
+
+
+.. _admin-capabilities-general_import:  
+
+General Import/Action Operations
+--------------------------------
+
+This capability enables common operations for import and export, such as the ability to load the error log. It is 
+recommended to set this capability to *"System"* if either *"Export Entities"* or *"Import Entities"* is 
+set to *"System"*. 
```

Comment:
Edit: This capability enables common operations for import and export, such as the ability to load the error log. It is recommended that you set this capability to *"System"* if either *"Export Entities"* or *"Import Entities"* is also set to *"System"*.